testing/sdk_install.py

```diff
@@ -13,7 +14,7 @@
 import sdk_plan
 import sdk_utils
 
-from retrying import retry
+log = logging.getLogger(__name__)
```

Comment:
Sorry, my bad. And yes, we should add flake8 or something similar for the python code.